src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -627,38 +614,22 @@ else if (entity instanceof EntityMinecart)
         }
     }
 
-    private boolean handleMaterials(Block block, IBlockState metadata)
+    private boolean handleMaterials(Block block, IBlockState blockState)
     {
         //Breaking blocks doesn't require taking materials from the citizens inventory
-        if (block != Blocks.air)
-        {
-            if (isBlockFree(block, block.getMetaFromState(metadata)))
-            {
-                return true;
-            }
-
-            ItemStack stack = new ItemStack(Item.getItemFromBlock(block), 1, block.damageDropped(metadata));
-
-            if (stack.getItem() == null)
-            {
-                stack = new ItemStack(block);
-            }
-
-            if (stack.getItem() == null)
-            {
-                stack = new ItemStack(BlockUtils.getItemFromBlock(block));
-            }
+        if (block == Blocks.air) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)